bin/auth-api/src/routes/workspace.routes.ts

```diff
@@ -211,7 +211,7 @@ router.get("/workspace/:workspaceId/members", async (ctx) => {
   ctx.body = members;
 });
 
-router.post("/workspace/:workspaceId/membership", async (ctx) => {
+automationApiRouter.post("/workspace/:workspaceId/membership", async (ctx) => {
   const { authUser, workspace } = await authorizeWorkspaceRoute(ctx, [
```

Comment:
I believe this will work, but we should make certain API tokens from non-owners can't take action here--i don't think we've had occasion to test that before.